src/variables/variable_common.jl

```diff
@@ -52,9 +52,33 @@ function add_variable!(
             ind => _variable(m, name, ind, lb, ub, bin, int)
             for ind in indices(m; t=vcat(history_time_slice(m), time_slice(m)))
         )
+    if use_representative_days(model=m.ext[:instance]) == :use_representative_days_w_seasonal_storage
```

Comment:
Is the spine_opt_template.json updated at this point? I think we'd need to add the use_representative_days parameter together with the possible 'enumerated' values. Which others would we have apart from use_representative_days_w_seasonal_storage? is the `w_seasonal_storage` part really necessary? Maybe it could be just a true/false thing.

Reply:
Yes, I think we could have a boolean parameter `use_representative_days_w_seasonal_storage`. I think the `w_seasonal_storage` doesn't look really nice, but I think it'd be important to indicate it here. (For "normal" representative days we don't need the whole mapping thing).
Alternatively, we can also trigger the mapping by the existence of the `ordering_periods` parameter, I think. Then we wouldn't need the `use_representative_days_w_seasonal_storage` parameter at all.